spec/services/vot/parser_spec.rb

```diff
@@ -48,16 +48,19 @@
         result = Vot::Parser.new(vector_of_trust:).parse
 
         expect(result.component_values.map(&:name).join('.')).to eq('C1.C2.P1.Pe')
```

Comment:
Idle thought that `component_values.map(&:name).join('.')` might be a good `to_s` implementaton

Reply:
or maybe even `to_vot`

Reply:
I just pushed a change that uses the existing `expanded_component_values` method. This was a pattern I copied form all of the other tests in this file so I also applied it to those.